staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go

```diff
@@ -387,12 +388,13 @@ func (p *Patcher) deleteAndCreate(original runtime.Object, modified []byte, name
 		return modified, nil, err
 	}
 	// TODO: use wait
-	if err := wait.PollImmediate(1*time.Second, p.Timeout, func() (bool, error) {
+	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, p.Timeout, true, func(ctx context.Context) (bool, error) {
```

Comment:
What is the reason of taking out from `if err` statement?.

Reply:
No particular reason. I can use the `if err` statement again if you'd prefer

Reply:
yes, please revert it back to `if err`

Reply:
reverted :)
btw what about this [line](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/apply/patcher.go#L389) should it be there?